java/src/main/java/ai/rapids/cudf/Scalar.java

```diff
@@ -373,6 +372,83 @@ public static Scalar listFromColumnView(ColumnView list) {
     return new Scalar(DType.LIST, makeListScalar(list.getNativeView(), true));
   }
 
+  /**
+   * Creates a null scalar of struct type.
+   *
+   * @param elementTypes data types of children in the struct
+   * @return a null scalar of struct type
+   */
+  public static Scalar structFromNull(HostColumnVector.DataType... elementTypes) {
+    ColumnVector[] children = new ColumnVector[elementTypes.length];
```

Comment:
Does it allow empty columns here?
I saw there is a check `columns.length == 0` in `structFromColumnViews`.

Reply:
Yes, it accepts empty columns. I've added corresponding test cases.